faq/views.py

```diff
@@ -15,38 +14,72 @@ def get_queryset(self):
         return Category.objects.prefetch_related('questions')
 
 
-class FAQAdminView(PermissionRequiredMixin, ListView):
-    model = Question
-    template_name = "faq/faq_admin.html"
-    context_object_name = "question_list"
+class FAQAdminPanelView(PermissionRequiredMixin, TemplateView):
+    template_name = 'faq/faq_admin_panel.html'
 
     def has_permission(self):
         return has_any_faq_permissions(self.request.user)
 
 
+class QuestionAdminView(PermissionRequiredMixin, ListView):
+    permission_required = ('faq.edit_question',)
+    model = Question
+    template_name = 'faq/admin_question_list.html'
+    context_object_name = 'questions'
+    queryset = Question.objects.order_by('title')
+
+
+class CategoryAdminView(PermissionRequiredMixin, ListView):
+    permission_required = ('faq.edit_category',)
```

Comment:
See the comment above.
```suggestion
    permission_required = ('faq.change_category',)
```